### DISCUSSION TITLE
[Feature] options of storage capacity between percentage and absolute size of immich data

### DISCUSSION THREAD

**Discussion by nevadax0120** [immich-app/immich#8488] [Feature Request]
### The feature

Please provide options to show the percentage and absolute size of immich data on the home page rather than always showing the full disk percentage.

Sometimes, I don't store the immich data independently.  In the other situation, I use s3 as the backend storage, so the disk size is unlimited/256PiB and the percentage is always 0%, which is useless.

<img width="269" alt="Screenshot 2024-04-03 at 7 56 22 PM" src="https://github.com/immich-app/immich/assets/137494677/b10c6ec6-0da1-434e-8a56-fe9295ba5b4a">

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This somehow got missed, but if it's showing 0B used then that's probably an issue with the underlying (v)fs rather than with Immich.
